pkg/networks/usernet/recoincile.go

```diff
@@ -0,0 +1,128 @@
+package usernet
+
+import (
+	"bytes"
+	"context"
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"time"
+
+	"github.com/lima-vm/lima/pkg/store"
+	"github.com/lima-vm/lima/pkg/store/dirnames"
+	"github.com/sirupsen/logrus"
+)
+
+func StartUsernet(ctx context.Context, name string) error {
```

Comment:
Seems to need flock

Reply:
Done. 
Moved the storage location of usernet files to <LIMA_HOME>/_networks/<name>/files